### PR TITLE
Update to 2.4.50

### PR DESCRIPTION
**Pull request by reporter4u**
This is a pull request for issue #49. I hope you can validate it and do reviews as soon as possible.

Thank you in advance.

### CODE REVIEW

**Comment by reporter4u** [comment]
I update the PR since this morning Apache Httpd mainstream fixed these CVE and published 2.4.50 release.
SECURITY: CVE-2021-41773
SECURITY: CVE-2021-41524
https://downloads.apache.org/httpd/CHANGES_2.4.50

File: httpd24u.spec [hunk 1]
```diff
@@ -824,6 +824,12 @@ exit $rv
 
 
 %changelog
+* Fri Oct 05 2021 Reporter4u <reporter4u@gnail.com> - 2.4.50-1
+- Latest upstream
+
+* Fri Oct 05 2021 Reporter4u <reporter4u@gnail.com> - 2.4.49-1
+- Latest upstream
```
Comment (carlwgeorge) [inline httpd24u.spec:831]: It's allowed to update directly from 2.4.48 to 2.4.50, you don't have to add a changelog entry for the version we're skipping.

**Review by carlwgeorge — CHANGES_REQUESTED** [review]
Overall this looks good, just drop the unneeded 2.4.49 changelog entry and I'll merge this.

File: httpd24u.spec (hunk 1 above)
Comment (reporter4u) [reply httpd24u.spec:828]: I removed 2.4.49 changelog.

**Review by carlwgeorge — APPROVED** [review]

**Pull request merged**